src/main.py

```diff
@@ -1,22 +1,26 @@
 import telegram
+import os
 from flask import Flask, request
 from jinja2 import Environment, FileSystemLoader
 
-# set current directory
+# Set current directory.
 ENV = Environment(loader=FileSystemLoader('.'))
+# Jinja template of the bot.
+TEMPLATE_PATH = './templates/template_message.j2'
 
-# get template
-baseline = ENV.get_template("./template_message.j2")
+# Load Jinja template.
+baseline = ENV.get_template()
 
 # flask
 app = Flask(__name__) 
 
 # telegram bot
 bot = telegram.Bot('TOKEN')
+chat_id = os.getenv('CHATID')
```

Comment:
@zufardhiyaulhaq probably we need to provide the env var in the K8s Job.

Reply:
should we provide the env var for telegram bot token to?

Reply:
yes.. I think better that way, we can get the env var from the secret we provided

Reply:
this is addressed as well..